### ISSUE TITLE
Request: Please introduce season pack downloading

### ISSUE THREAD

**Issue by qvazzler** [Flexget/Flexget#948]
Hi. Will keep it short but sweet.

Many private trackers group up previous seasons into season packs, and currently flexget has difficulty getting these.

Would it be possible to work towards enabling season pack downloading through functionality like backfill from discover?

Thanks.

**Comment by qvazzler** [comment]
For the record, this is what I have working to download season packs currently. Cannot be grouped with any other config tasks, as it alters the sequence mode for episodes. (S01E01 thrown out of the window)

```
  get_series_packs:
    priority: 2
    content_size:
      min: 4000
    regexp:
      reject:
        - \.[sS]\d\d[eE]\d\d\.
        - \.FiX\.
        - FASTSUB #French
        - VOSTFR #French
        - Subtitulado #Spanish
        - Special-Wicked #Special trailer episodes from Once Upon a Time
        - Magazine #No magazines on Arrow, thank you.
        - NLsubs
    content_filter:
      reject:
        - '*.avi'

    configure_series:
      from:
        trakt_list:
          username: '{{ secrets.trakt.username }}'
          account: '{{ secrets.trakt.account }}'
          list: 2-b-shows-b-get-backlog
          type: shows
          strip_dates: yes
        #listdir:
          #- /volume1/Disk1/Library/TV Shows
      settings:
        timeframe: 2 hours
        target: 720p hdtv
        identified_by: sequence
        sequence_regexp: \b[S][0]?(\d+)\b
        #sequence_regexp: (\d+)
        exact: yes
    manipulate:
      - title:
          replace:
            regexp: '(?<=\.[Ss]\d\d)[eE]01(?=\.)' # Remove E01 from each episode and turn it into a "Season entry"
            format: ''
      - title:
          replace:
            regexp: '.*([eE]0[^1]).*|.*[eE]([1-9]\d).*' # Remove any entry that is not the first episode of a season
            format: ''
    discover:
      what:
        - emit_series:
            from_start: yes
            backfill: yes
      from:
        - sceneaccess:
            username: '{{ secrets.sceneaccess.username }}'
            password: '{{ secrets.sceneaccess.password }}'
            gravity_multiplier: 200
            category:
              archive:
                - TV/Packs
      #ignore_estimations: no
    template:
      - transmit-series
      - transmit-manual-download
```

**Comment by liiight** [comment]
Pr is done, if you wanna help with testing... 
#1710

**Comment by wbern** [comment]
@liiight that's great! I will try to make time to change up my configurations. Simplifying them will be a welcome change.

**Comment by liiight** [comment]
Closed with #1710
